lib/arjdbc/vertica/adapter.rb

```diff
@@ -104,24 +111,6 @@ def bulk_insert(table_name, primary_key, sequence_name, column_names, data)
       execute(sql)
     end
 
-    def columns(table_name, name = nil)
```

Comment:
Is this method no longer required?

Reply:
It'll probably take me a while to remember the specifics, (i'm digging into it again now)

but basically this method no longer worked as written and instead should be using the parent method instead

https://github.com/jruby/activerecord-jdbc-adapter/blob/51-stable/lib/arjdbc/jdbc/adapter.rb#L224

In AR 4 the `new_column` method for the abstract_adapter was 
https://github.com/rails/rails/blob/4-2-stable/activerecord/lib/active_record/connection_adapters/abstract_adapter.rb#L388
```
def new_column(name, default, cast_type, sql_type = nil, null = true)
        Column.new(name, default, cast_type, sql_type, null)
      end
```

and in AR5 it is
https://github.com/rails/rails/blob/5-1-stable/activerecord/lib/active_record/connection_adapters/abstract_adapter.rb#L485

```
      def new_column(name, default, sql_type_metadata, null, table_name, default_function = nil, collation = nil)
        Column.new(name, default, sql_type_metadata, null, table_name, default_function, collation)
      end
```


Instead of trying to handle the creation and passing of the `sql_type_metadata`, i tried using the default `columns` method provided by the parent class and it was already wired up to pass through the arguments in the right way.

Reply:
Okay after fixing the instantiation of the VerticaColumn again i think we're at a better spot, but i still am unsure of how to exactly verify that the end schema/state of the VerticaColumn is what it needs to be